frontend/webapp/cypress/e2e/action-testing.cy.ts

```diff
@@ -0,0 +1,20 @@
+describe('Action Addition Testing', () => {
+    it('Add an Action', () => {
+        cy.visit('/')
+        cy.get('[data-cy="menu-Actions"]').should('exist').click()
+        cy.url().should('eq', `${Cypress.config('baseUrl')}/actions`);
```

Comment:
```suggestion
const baseUrl = Cypress.config('baseUrl');
        cy.url().should('eq', `${baseUrl}//actions`);
```
calling only once to the cypress API and using it in all the places